src/plugins/intel_npu/src/al/src/config/options.cpp

```diff
@@ -0,0 +1,136 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "intel_npu/al/config/options.hpp"
+
+using namespace intel_npu;
+using namespace ov::intel_npu;
+
+//
+// register
+//
+
+void intel_npu::registerOptions(OptionsDesc& desc, compilerVersion compilerVer) {
+    // reset options first
+    desc.reset();
+
+    // register
+    desc.add<PERFORMANCE_HINT>(compilerVer);
```

Comment:
The compilerVer is get from driver, shall we just pass it to desc to reduce times to pass arguments compilerVer?  And from here, old add() is not used any more, right?

Reply:
Will modify, thanks!